app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1> Admin Dashboard </h1>
+
+<h2> Invoice ID: <%= @invoice.id %> </h2>
+<p> Status: <%= @invoice.status %> </p>
+<p> Created at: <%= @invoice.created_at.strftime("%A, %B %d, %Y") %> </p>
+<h3> Customer: <%= "#{@invoice.customer.first_name} #{@invoice.customer.last_name}" %> </h3>
```

Comment:
To make make this a bit cleaner you could instantiate customer in the controller and then only have `customer.first_name/last_name` however this works!